Intro Story Draft One Complete.

dapperAuteur opened the pull request:
Intro Story Draft One Complete. Doesn't Comletely Off Screen. I Need Help With That.

dapperAuteur committed:
Intro Story Draft One Complete.

Intro Story Draft One Complete. Doesn't Comletely Off Screen. I Need Help With That.

mmenavas commented:
@dapperAuteur You'll want to play with the transition values in the code below in introduction.less

```
#introduction > .welcome {
    position: fixed;
    left: 10%;
    top: 100%;
    width: 80%;
    font-size: 3em;
    color: gold;
    transition: top 20s linear;
}
```

and also with the top value on in the code below in the same file

```
#introduction.welcome > .welcome {
    top: -3000px;
}
```

and finally update the timeout value for the timeout function that show the Begin message in introduction.js

```
        playAnimation: function(){
            var _this = this;
            setTimeout(function () {
                _this.$el.find('#introduction').addClass('play');
                setTimeout(function () {
                    _this.$el.find('#introduction').addClass('explosion');
                    setTimeout(function () {
                        _this.$el.find('#introduction').addClass('welcome');
                        setTimeout(function () {
                            _this.$el.find('#introduction').addClass('begin');
                        }, 25000);
                    }, 1000);
                }, 3000);
            }, 500);   
        }
```

Let me know if you have any questions =)

dapperAuteur committed:
Shortened Story To Fit Time Constraint

I couldn't figure out how to change the scroll to fit the story so I shortened the story to fit the time. I think the story is better. Let me know your thoughts.

mmenavas commented:
@dapperAuteur Thank you for contributing with the story. I'm merging this pull request but I'm also creating a new issue because the scrolling text has not been optimized for small screens. Next time, don't let you creativity be constrained by the technology. We can always fix CSS and JS issues separately.
